### ISSUE TITLE
Add missing URLs to landing page and update key dates

### ISSUE THREAD

**Issue by marcellmueller** [bcgov/cas-registration#668]
#### Describe the task

This is a follow up ticket to #163 since the URLs weren't ready on time.

Notes about the URLs are in the [Wireframe](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=2688-6375&mode=design&t=y7smsRJhRtzkRitv-4).

The Key Dates table should also be updated according to the latest design above since we postpone the prod release date.

#### Acceptance Criteria

- [x] BC OBPS Program Website URL
- [ ] Carbon Tax Exemption URL (please see comments below)
- [x] Detailed Guidance URL
- [x] Update Key Dates in the table

**Comment by Dianadec** [comment]
Once we have the info, we can bring into the "Isolated Simple Tasks"

**Comment by nanyangpro** [comment]
The Key Dates table is updated as advised by @Maralsotoudehnia. I've also included it in the AC.

We're still waiting for the URLs. Once we get them, I'll include them in the design and inform the developers for implementation.

**Comment by nanyangpro** [comment]
Maral advised that "FIN's carbon tax exemption link won't be ready until late March, so I suggest removing it from the page for now."
Please remove the paragraph with the carbon tax exemption link for the MVP prod release.

**Comment by nanyangpro** [comment]
We got the links, this ticket is unblocked now! :)
